teacher.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'dbc_community'
+
+class Teacher < DBC
+  attr_reader :target_raise , :salary
+  def initialize(options={})
```

Comment:
Same here. 
This reads as "a teacher is a kind of DBC" which doesn't sound quite right. Student and Teacher are both people. How could we represent this with our hierarchy?